src/qweb/qweb.ts

```diff
@@ -418,6 +431,11 @@ export class QWeb extends EventBus {
         }
         text = text.replace(whitespaceRE, " ");
       }
+      if (this.translateFn) {
+        if ((node.parentNode as any).getAttribute("t-translation") !== "off") {
```

Comment:
shouldn't t-translation directive apply on the whole subtree, not only direct children?